src/libexpr/primops.cc

```diff
@@ -4857,7 +4857,10 @@ void EvalState::createBaseEnv()
     addConstant("__nixPath", v, {
         .type = nList,
         .doc = R"(
-          The value of the [`nix-path` configuration setting](@docroot@/command-ref/conf-file.md#conf-nix-path): a list of search path entries used to resolve [lookup paths](@docroot@/language/constructs/lookup-path.md).
+          A list of search path entries used to resolve [lookup paths](@docroot@/language/constructs/lookup-path.md).
+          Its value is primarily determined by the [`nix-path` configuration setting](@docroot@/command-ref/conf-file.md#conf-nix-path), which can be
```

Comment:
```suggestion
          Its value is primarily determined by the [`nix-path` configuration option](@docroot@/command-ref/conf-file.md#conf-nix-path), which can be
```

or do we accept _setting_ as a (potentially redundant) term for the value and/or value definitions of an option?

Reply:
That was the idea, but why would that be redundant? Option gives you the ability to set something, setting is the (implicit, if you don't set it yourself) result of that.

Reply:
It's only redundant because _option_ would get the job done.

Unless we're talking about the functioning of the configuration system, this level of detail is not necessary and we could make do with fewer words, and perhaps fewer opportunities for irrelevant thoughts.

Analogously, in the context of the Module System it's normal to refer to an _option_ even if it's really only about the `value` of it. Settings is only used in the context of RFC42 there, which is a bit arbitrary, but makes the term part of the "user" domain instead of the Module System proper.

Reply:
I'm fine either way.